### ISSUE TITLE
Add a button to expand all cards

### ISSUE THREAD

**Issue by brittanyrw** [brittanyrw/emojipages#33]
Add a button on the homepage that will toggle showing and hiding the song name. This button will allow people to show all names if they just want to browse vs having to click on each card to see the name.

When you click on the button it should toggle, ie if I click and nothing is expanded, everything will expand and then when I click again, everything should hide again.

Let us know if you want assistance with the design for this! Otherwise, feel free to come up with something on your own.

**Comment by burianovata** [comment]
Hey! I would like to help you. Please tell where the button should appear? I guess between the header and books section?

**Comment by brittanyrw** [comment]
@burianovata Yes, it can go there if you like! I am pretty flexible with this. 🙂 It could go in the top right corner, inside of the header section...etc. Just as long as people can easily use it.

You can see an example of how someone implemented this on [EmojiScreen](https://emojiscreen.com). It's the purple button with the eye icon. They decided to but it in the bottom right of the screen.

<img width="531" alt="Screen Shot 2019-09-26 at 7 38 07 AM" src="https://user-images.githubusercontent.com/23530405/65685844-f4b40000-e030-11e9-8e48-c7cc934cf3e1.png">

**Comment by Jxshvh** [comment]
Since previous user has not yet forked the repo I'll take a look at it, Sounds like a very interesting and fun project and I'd love to contribute.
